client/reducers/events.js

```diff
@@ -67,11 +68,19 @@ const eventsReducer = (state=initialState, action) => {
             return {
                 ...state,
                 showEventDetails: action.payload,
+                showEventForm: true,
             }
         case 'CLOSE_EVENT_DETAILS':
             return {
                 ...state,
                 showEventDetails: null,
+                showEventForm: false,
+            }
+        case 'DELETE_EVENT':
+            return {
+                ...state,
+                showEventDetails: action.payload,
```

Comment:
I need the event object with detail _planning, but I do not want the event form to be open. Hence the two options.
Please suggest if you would prefer any other approach.